src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,360 @@
+         *         + "|____/ \\__,_|_|\\_\\___|\n";
+         * System.out.println("Hello from\n" + logo);
+         */
+
+        introMessage();
+        try {
+            getTasksFromFile();
+        } catch (FileNotFoundException f) {
+            System.out.println("File not found");
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        System.out.println("____________________________________________________________");
+
+        //Scanner in
+        Scanner in = new Scanner(System.in);
+        String userInput;
+
+        //Check user input and add/modify accordingly
+        do {
+            userInput = in.nextLine();
+            String[] userInputs = userInput.split("\\s+");
+            if (!userInput.equals("bye")) { //this line is so that bye is not 'added'
+                System.out.println("____________________________________________________________");
+                try {
+                    switch (userInputs[0]) {
+                    case "todo":
+                        addTodoToList(userInputs, true);
+                        break;
+                    case "deadline":
+                        addDeadlineToList(userInputs, true);
+                        break;
+                    case "event":
+                        addEventToList(userInputs, true);
+                        break;
+                    case "list":
+                        listTasks();
+                        break;
+                    case "done":
+                        markTaskAsDone(userInputs);
+                        break;
+                    case "delete":
+                        deleteTask(userInputs);
+                        break;
+                    default:
+                        System.out.println("OOPS!!! I'm sorry, but I don't know what that means :-(");
+                        break;
+                    }
+                } catch (DukeException e) {
+                    System.out.println(e);
+                } catch (ArrayIndexOutOfBoundsException e) { //done and delete
+                    System.out.println("OOPS!!! The command needs to be followed by a task number." );
+                } catch (IndexOutOfBoundsException e) {
+                    System.out.println("OOPS!!! The specified task does not exist. There are only " +
+                            userInputList.size() + " tasks");
+                } catch (IOException e) {
+                    System.out.println("OOPS!!! Something went wrong." );
+                    e.printStackTrace();
+                }
+                System.out.println("____________________________________________________________");
+            }
+        } while (!userInput.equals("bye"));
+
+        exitDuke();
+    }
+
+    public static void getTasksFromFile() throws IOException {
+        File f = new File("./data/duke.txt");
+        boolean mkdir = new File("./data").mkdir();
+        boolean result = f.createNewFile();
```

Comment:
ALso, I think, a better way is to test the presence of the file explicitly, and create one if it is not present